misc/cleanup-dash.py

```diff
@@ -170,6 +170,9 @@ def drop_some_internal_elements(dashboard):
             del dashboard['__inputs']
         if '__requires' in element:
             del dashboard['__requires']
+        for list_index, listelement in enumerate(dashboard['templating']['list']):
+            if 'current' in listelement:
+                listelement['current'] = {}
```

Comment:
Isn't it more clear `dashboard['templating']['list'][index] = {}` ?

Reply:
Dropping of whole section list is leading to next error.
So I have rewritten it to
dashboard['templating']['list'][index]['current'] = {}

![Screenshot_20190311_152430](https://user-images.githubusercontent.com/25184101/54127355-3e7d2600-4412-11e9-847c-9987bc3a7170.png)